src/api/cardsApi.js

```diff
@@ -5,7 +5,13 @@ function handleResponse(response) {
   throw new Error(response.text());
 }
 
+function invalid(card) {
+  return (!card.text || !card.text.length || card.columnId === null || card.columnId === undefined);
```

Comment:
You need to trim the text before you check the length, cause I can send an empty space and send the request properly.